### DISCUSSION TITLE
Branch creation policy

### DISCUSSION THREAD

**Discussion by romacdon** [openthread/openthread#9847] [Q&A] [answered]
@jwhui what's the policy for creating branches in the openthread/ot-efr32 repo?

**Comment by jwhui** [comment, marked as answer]
For default and release branches, we require the following:

- Require a pull request before merging. Under additional settings:
  - Require approvals (select at least 1-2 as the required number of approvals before merging).
- Require status checks to pass before merging. Under additional settings:
  - Require branches to be up to date before merging (may be problematic for busy repos).
- Block force pushes
- Restrict deletions
- Limit any bypass actors to those that are strictly necessary (i.e. break glass).
